docs/common/low_power.md

```diff
@@ -41,11 +38,11 @@ CONFIG_LTE_PSM_REQ_RPTAU_SECONDS=1800  # Periodic TAU: 30 minutes
 CONFIG_LTE_PSM_REQ_RAT_SECONDS=60      # Active Time: 1 minute
 ```
 
-It's recommended to configure a Periodic TAU timer longer than the update interval of the application to avoid synchronization with the network between updates.
+It is recommended to configure a Periodic TAU timer longer than the update interval of the application to avoid synchronization with the network between updates.
```

Comment:
```suggestion
It is recommended to configure a periodic TAU timer longer than the update interval of the application to avoid synchronization with the network between updates.
This is to avoid unnecessary network traffic between data sending.
```